packages/discovery-provider/src/tasks/entity_manager/entities/playlist.py

```diff
@@ -667,6 +679,9 @@ def process_playlist_data_event(
     playlist_record.updated_at = block_datetime
     playlist_record.metadata_multihash = metadata_cid
 
+    if is_ddex_signer(params.signer):
```

Comment:
note to self: still need to list out all ddex keypairs in env variable.  We might want to either move this to code, or remove the requirement to know the keypair ahead of time?